src/scripts/dad-joke.js

```diff
@@ -5,6 +5,7 @@ const {
   helpMessage,
   stats: { incrementStats },
 } = require("../utils");
+const sample = require("../utils/sample");
```

Comment:
To your question in the PR, this seems fine! I don't think it's my ideal case, but it's better than fighting with the tests forever. Maybe someday we can come back and find a way to make the tests work properly, but this is totally within the realm of "good enough," which is all anything needs to be. 😂